src/vsr/replica.zig

```diff
@@ -4557,16 +4656,56 @@ pub fn ReplicaType(
                 return;
             }
 
-            const next = @mod(self.replica + 1, self.replica_count);
-            if (next == self.primary_index(message.header.view)) {
-                log.debug("{}: replicate: not replicating (completed)", .{self.replica});
-                return;
-            }
+            const next = next: {
+                if (!self.standby()) {
+                    // First, replicate to the next active replica.
+                    const next_replica = @mod(self.replica + 1, self.replica_count);
+                    if (next_replica == self.primary_index(message.header.view)) {
```

Comment:
Note for tomorrow’s self: flip this branch so that the source code order correspond to how the replication actually is going.